.github/workflows/sanitizers.yml

```diff
@@ -68,7 +68,9 @@ jobs:
       working-directory: ${{env.BUILD_DIR}}
       run: > 
         ${{ matrix.compiler.cxx == 'icpx' && '. /opt/intel/oneapi/setvars.sh &&' || ''}}
-        ctest --output-on-failure
+        ctest --output-on-failure -E "umf-provider_os_memory$"
```

Comment:
I think that this:
`GTEST_FILTER="-*umfProviderTest.alloc_page64_align_0*" ctest --output-on-failure`
should be enough. See: https://google.github.io/googletest/advanced.html#running-a-subset-of-the-tests

Reply:
Done